prime-router/settings/prod/0016-az-updates.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Our rule is that settings due to be uploaded to production get put in settings/prod/nnnn-<NAME>.yml.

They get reviewed by a human as part of the PR process, then they get inserted into production manually via a call to 
`./prime receiver put --input settings/prod/0016-az-updates.yml`